src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.data.tag;
+
+public class Tagging {
+	
+	private static StringBuilder fullTaggingList = new StringBuilder (); 
```

Comment:
Putting the stringBuilder here doesn't follow the diagram, you should declare it as an object in `AddressBook` class, check #1618.